components/composer/src/Composer.svelte

```diff
@@ -210,10 +208,9 @@
     showDatepicker = false;
   };
 
-  const handleFilesChange = async () => {
-    if (!fileSelector?.files) {
-      return;
-    }
+  async function handleFilesChange(fileChanged: any) {
```

Comment:
`fileChanged` should probably be an Event type?

Reply:
Oh yeah that was temp - I didn't know what it was, thanks!